src/App.js

```diff
@@ -1,23 +1,54 @@
 import logo from './logo.svg';
 import './App.css';
+import {BrowserRouter, Routes, Route, Link} from 'react-router-dom';
+import HomePage from './components/HomePage';
+import AllBeers from './components/AllBeers';
+import BeerDetails from './components/BeerDetails';
+import RandomBeer from './components/RandomBeer';
+import NewBeer from './components/NewBeer';
+import { useState } from 'react';
+import axios from "axios";
+import { useEffect } from 'react';
+
 
 function App() {
+  const [beers, setBeers] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(process.env.REACT_APP_API_URL)
+      .then((response) => {
+        setBeers(response.data);
+      })
+      .catch((e) => console.log(e));
+  }, []);
+
+
+  const getContributorName = (contributor) => {
+    const startIndex = contributor.indexOf("<");
```

Comment:
The guard that I was referring

```suggestion
    const startIndex = contributor?.indexOf("<");
```